### PR TITLE
Schedule daily run of scalability benchmark

### LINKED ISSUE #366: Schedule scalability scenario for currently defined scenarios
We would like to run a client credential grant scalability scenario with some good input parameters to the kcb.sh script on a daily basis on weekdays and capture the mean throughput and mean response time of the endpoints part of the client credential grant flow and write it out to a csv file on a branch where it won't be subject to TTL for archived files.

As part of this task, we also will start a discussion within the team to find out the best input parameters for attributes such as,

- warm-up
- ramp-up
- measurement
- user-think-time
- sla-error-percentage
- sla-mean-response-time
- clients-per-realm
- realms

### PR DESCRIPTION
**Pull request by mhajas**
Closes #366

### CODE REVIEW

**Comment by mhajas** [comment]
@ahus1 @kami619 I am not sure what exactly should be part of this issue. Should I add also some csv file with results to kcb.sh? Or this will be done as part of #373

**Comment by kami619** [comment]
is it possible to plot the data from the archived jsons in a small summary csv based on the gatling runs ? we cam then use the the other ticket to store this data elsewhere

**Comment by ahus1** [comment]
I'd argue that creating a CSV or JSON different from Grafana is outside of the scope of this issue, and should be part of #373.

**Comment by mhajas** [comment]
I agree with @ahus1 here, as part of this issue, we don't know what format we need, so I would rather postpone it to #373 We can then split the issue into more if needed.

**Comment by kami619** [comment]
> I agree with @ahus1 here, as part of this issue, we don't know what format we need, so I would rather postpone it to #373 We can then split the issue into more if needed.

ok, my attempt for a requirement creep-in didn't work ;) I am fine with it to be part of #373

File: .github/workflows/keycloak-scalability-benchmark-on-schedule.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+jobs:
+
+  checkout:
+    name: ROSA Scheduled Create cluster
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run ClientSecret scalability benchmark
+        run: |
+          gh workflow run -R keycloak/keycloak-benchmark keycloak-scalability-benchmark.yml \
+            -f "scenarioName=authentication.ClientSecret"
```
Comment (kami619) [inline .github/workflows/keycloak-scalability-benchmark-on-schedule.yml:16]: do we want to define this for ClientSecret scenario as well ? `-f "numberOfEntitiesInRealm=100000"`

Comment (kami619) [reply .github/workflows/keycloak-scalability-benchmark-on-schedule.yml:16]: true, but I only see the `numberOfEntitiesInRealm` defined for AuthorizationCode scenario and not for ClientSecret., or am I missing something ? @mhajas

Comment (mhajas) [reply .github/workflows/keycloak-scalability-benchmark-on-schedule.yml:16]: `numberOfEntitiesInRealm` defaults to 10000, so it does not need to be added for ClientSecret. Do we want to add it too? For example, if the default changes?

**Pull request merged**